Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```
@@ -5339,6 +5356,12 @@ void WebGLRenderingContextBase::texImageImpl(TexImageFunctionID functionID, GCGL
 void WebGLRenderingContextBase::texImage2DBase(GCGLenum target, GCGLint level, GCGLenum internalFormat, GCGLsizei width, GCGLsizei height, GCGLint border, GCGLenum format, GCGLenum type, GCGLsizei byteLength, const void* pixels)
 {
 #if USE(ANGLE)
+    unsigned bytesRequired = 0;
+    bool ok = computeImageSizeInBytes(TexImageDimension::Tex2D, format, type, width, height, 1, &bytesRequired);
+    ASSERT_UNUSED(ok, ok, "Parameters should already have been validated!");
+    // FIXME: ANGLE robust API uses signed int for byte count.
+    ASSERT(isInBounds<GCGLsizei>(bytesRequired));
```

Should we throw an error if our `unsigned` value is out of the signed bounds?